tests/integration.rs

```diff
@@ -102,6 +111,7 @@ fn clean_and_parse(target: &TempDir, args: &[&str]) -> Result<u64> {
     Ok(cleaned)
 }
 
+#[allow(clippy::uninlined_format_args)]
```

Comment:
@jyn514 actually, I added this because the _git hooks_ wouldn't let me push without it, have you solved this in the last PR?

Reply:
ah gotcha - you can use `--no-verify` to bypass the git hook, or rebase over master where I've fixed it.

I wonder if I should get rid of clippy in the pre-push hook and only run rustfmt instead :thinking: